### DISCUSSION TITLE
Possible to simulate indoor fan rpm?

### DISCUSSION THREAD

**Discussion by TechnicGuy1** [revk/xESP32-Faikout#356] [Ideas]
Hello,

i'm using a Daikin ATXF50A/ARXF50A with faiking, yes it's running fine :)
But modified it to be a water heat pump, so use indoor a plate heat exchanger, only "problem" is, that the indoor fan must run.
Is there a  way to simulate the rpm signal with faikin? Would be very great :)

Greetings the TechnicGuy1

**Comment by revk** [comment]
There is not a message for that, I don't think, and we don't have a spare GPIO available (well maybe but it will be unbuffered 3.3V).

**Reply by TechnicGuy1** [reply]
ok hoped, there is a message that could be created. The Outdoor Unit starts, after some seconds it shut down. Have you information from the rpm signal, how it should me to simulate it external?
